graph-courses.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+import argparse
+import collections
+import operator
+import calendar as cal
+from Class import *
+from ConvertParse import *
+
+reload(sys)
+sys.setdefaultencoding('utf-8')
+
+args = argparse.ArgumentParser()
+args.add_argument('filename', nargs='+')
+args = args.parse_args()
+courses = collections.defaultdict(list)
+list_course = collections.defaultdict(dict)
```

Comment:
Let's move these declarations below, to just before we use them for the first time.